packages/babel-traverse/test/scope.js

```diff
@@ -46,7 +46,7 @@ describe("scope", function () {
       assert.ok(getPath("({ x: 1 })").get("body")[0].get("expression").isPure());
     });
 
-    test("label", function () {
+    it("label", function () {
```

Comment:
Jest supports either `test` or `it` so this is unnecessary (unless you like `it` better 👍)